x-pack/plugins/maps/common/content_management/v1/cm_services.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+import type { ContentManagementServicesDefinition as ServicesDefinition } from '@kbn/object-versioning';
+
+const apiError = schema.object({
+  error: schema.string(),
+  message: schema.string(),
+  statusCode: schema.number(),
+  metadata: schema.object({}, { unknowns: 'allow' }),
+});
+
+const referenceSchema = schema.object(
+  {
+    name: schema.maybe(schema.string()),
+    type: schema.string(),
+    id: schema.string(),
+  },
+  { unknowns: 'forbid' }
+);
+
+const referencesSchema = schema.arrayOf(referenceSchema);
+
+const mapAttributesSchema = schema.object(
+  {
+    title: schema.string(),
+    description: schema.maybe(schema.string()),
+    mapStateJSON: schema.maybe(schema.string()),
+    layerListJSON: schema.maybe(schema.string()),
+    uiStateJSON: schema.maybe(schema.string()),
+  },
+  { unknowns: 'forbid' }
+);
+
+const mapSavedObjectSchema = schema.object(
```

Comment:
Aside from `mapAttributesSchema`, this looks like a generic saved objects schema